examples/Cargo.toml

```diff
@@ -0,0 +1,83 @@
+[package]
+name = "webrender-examples"
+version = "0.1.0"
+authors = ["Zakor Gyula <gyula.zakor@h-lab.eu>"]
+license = "MPL-2.0"
+repository = "https://github.com/servo/webrender"
+
+[[bin]]
+name = "alpha_perf"
```

Comment:
are the bin paths required here? I'd assume it can figure this out automatically

Reply:
Yes, without these we get an error if trying to run e.g. the `alpha_perf` example with `cargo run --bin alpha_perf`:
```
error: failed to parse manifest at `C:\Users\zakorgy\work\webrender\examples\Cargo.toml`

Caused by:
  no targets specified in the manifest
  either src/lib.rs, src/main.rs, a [lib] section, or [[bin]] section must be present
```